api/openapi-spec/swagger.json

```diff
@@ -424,11 +424,19 @@
           "type": "string"
         },
         "status": {
-          "description": "Status of the condition, one of True, False, Unknown.",
+          "description": "Status of the condition, one of True, False, Unknown.\n\nPossible enum values:\n - `False`:\n - `True`:\n - `Unknown`:",
+          "enum": [
+            "False",
+            "True",
+            "Unknown"
+          ],
           "type": "string"
         },
         "type": {
-          "description": "Type of the condition.",
+          "description": "Type of the condition.\n\nPossible enum values:\n - `AllEncodingVersionsEqual`: Indicates that encoding storage versions reported by all servers are equal.",
```

Comment:
Are you capitalizing after the semicolon? I'm not sure it should be

Reply:
Likely a mistake from the original author.